### PR TITLE
Familiarity sections and some quick links

### LINKED ISSUE #230: Sections for new and expert users

### LINKED ISSUE #241: Add back to table of content
Like other UG's [Back to Table of Contents](https://ay2324s1-cs2103t-t17-2.github.io/tp/UserGuide.html#table-of-contents)

### PR DESCRIPTION
**Pull request by drustanyjt**
# New And Experienced Section

Closes #230. Added sections for new and experienced users under guide usage:

Here's what the section looks like on the website.

image

# Quick Links To Top

Also added quick links to return to top of the page. Using CSS styles, the markdown file can be configured to show different messages depending on if the code is meant to be used as a website or as a printed document. Closes #241.

In this case:
- Print: Show "Back To Table of Contents"
- Generated: Show "Back To Top"

Example on website:

image

Example on printed UG:

image

Also note that the website has no Table of Contents header:
image

While the printed one does:
image

### CODE REVIEW

**Comment by drustanyjt** [comment]
This does not fix #236.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -71,6 +73,26 @@ Throughout this guide, you'll also encounter various text styles. Here's what th
...
+
+1. To get started, head over to the [Quick Start](#quick-start) section to set up ClinicMate on your computer.
+2. Once you've set up ClinicMate, go through the [Quick Tour](#TODO) to discover key features of ClinicMate that you'll use frequently.
+3. If you're keen to explore ClinicMate by yourself, take a look at the [Existing Features](#existing-features) to dive deep into ClinicMate's functionalities.
+
+## Seasoned ClinicMate User?
+
+Welcome back! We're thrilled to have you with us again.
+
+1. If you want a reminder on how to use a feature, head over to the [Existing Features](#existing-features) section to get **detailed instructions** for each feature and **examples** for you can learn from.
```
Comment (jiahui0309) [inline docs/UserGuide.md]: can consider "for you to learn from" or "that you can learn from"?

**Review by jiahui0309 — CHANGES_REQUESTED** [review]
A minor comment!

File: docs/UserGuide.md (hunk 1 above)
Comment (drustanyjt) [reply docs/UserGuide.md]: ok will change!

**Comment by drustanyjt** [comment]
@jiahui0309 have fixed the typo, could you help to look again?

**Review by jiahui0309 — APPROVED** [review]
LGTM

**Pull request merged**